### PR TITLE
Proper docker build fix

### PR DESCRIPTION
**Pull request by nirvn**
This fix is needed to allow for docker builds to work properly. An initial fix (the one that always removed the destination SRC_FOLDER) turned out to have broken release on github. I already reverted the change on release-3_1, this PR is fixing it properly on master.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 7e68c035aad36bed2116283f4c922c90b4a5c655: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4838-6853821f080043d4888dafb2cc4cad57bdc1bf3e-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-neon-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4838-6853821f080043d4888dafb2cc4cad57bdc1bf3e-arm-neon-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4838-6853821f080043d4888dafb2cc4cad57bdc1bf3e-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4838-6853821f080043d4888dafb2cc4cad57bdc1bf3e-x86-android.apk)
- [**all access arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4838-6853821f080043d4888dafb2cc4cad57bdc1bf3e-arm64-android.apk)
- [all access arm-neon-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4838-6853821f080043d4888dafb2cc4cad57bdc1bf3e-arm-neon-android.apk)
- [all access x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4838-6853821f080043d4888dafb2cc4cad57bdc1bf3e-x64-android.apk)
- [all access x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4838-6853821f080043d4888dafb2cc4cad57bdc1bf3e-x86-android.apk)
</details>

File: cmake/Package.cmake [hunk 1]
```diff
@@ -83,8 +83,10 @@ if(ANDROID AND ANDROIDDEPLOYQT_EXECUTABLE)
     set(ANDROID_TEMPLATE_FOLDER "${CMAKE_BINARY_DIR}/android-template")
     file(COPY ${CMAKE_SOURCE_DIR}/platform/android/ DESTINATION ${ANDROID_TEMPLATE_FOLDER}/)
     set(SRC_FOLDER "${ANDROID_TEMPLATE_FOLDER}/src/ch/opengis/${APP_PACKAGE_NAME}")
-    file(REMOVE_RECURSE ${SRC_FOLDER}) # remove any pre-existing content
-    file(RENAME "${ANDROID_TEMPLATE_FOLDER}/src/ch/opengis/qfield" ${SRC_FOLDER})
+    if (NOT APP_PACKAGE_NAME EQUAL "qield")
```
Comment (m-kuhn) [inline cmake/Package.cmake:86]: Typo?

Comment (nirvn) [reply cmake/Package.cmake:86]: @m-kuhn , you must have missed the memo about the app renamed to Qield? ;-P

Comment (nirvn) [inline cmake/Package.cmake:86]:
```suggestion
    if (NOT APP_PACKAGE_NAME EQUAL "qfield")
```

Comment (m-kuhn) [reply cmake/Package.cmake]: Indeed missed that 😅

One more thing: `STREQUAL` would be correct here, `EQUAL` is optimized for numbers.

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**